README.md

```diff
@@ -1,5 +1,278 @@
+
+Let the pre-constructed row values be:
+```kotlin
+val header = textRow(textCell(faker.getTitle(1)),
+                     textCell(faker.getTitle(1)),
+                     textCell(faker.getTitle(1)))
+val divider = textDivider(3)
+val row1 = textRow(textCell(faker.getWords(5),
+                            faker.getWords(2)),
+                   textCell(faker.getWords(2)),
+                   textCell(faker.getWords(3),
+                            faker.getWords(2),
+                            faker.getWords(1)))
+val row2 = textRow(textCell(faker.getWords(5)),
+                   textCell(faker.getWords(2),
+                            faker.getWords(2)),
+                   textCell(faker.getWords(3),
+                            faker.getWords(2),
+                            faker.getWords(1)))
+val row3 = textRow(textCell(faker.getWords(5),
+                            faker.getWords(2),
+                            faker.getWords(1)),
+                   textCell(faker.getWords(2),
+                            faker.getWords(2)),
+                   textCell(faker.getWords(3)))
+```
+
+Then, these values can be used in the following ways:
+
+##### Option 1
+
+```kotlin
+textTable(row1, row2, row3) {
+    withHeader(header)
+    withHeaderDivider(divider)
+}.println()
+```
+
+##### Option 2
+
+```kotlin
+textTable(listOf(row1, row2, row3)) {
+    withHeader(header)
+    withHeaderDivider(divider)
+}.println()
+```
+
+##### Option 3
+
+```kotlin
+textTable {
+    withBody(row1, row2, row3)
+    withHeader(header)
+    withHeaderDivider(divider)
+}.println()
+```
+
+##### Option 4
+
+```kotlin
+textTable {
+    addBody(row1)
+    addBody(row2)
+    addBody(row3)
+    withHeader(header)
+    withHeaderDivider(divider)
+}.println()
+```
+
+#### Java API
+
+#### Sample Output
+
+```
+In                                        |Mauris             |Omittam                     
+------------------------------------------|-------------------|----------------------------
+percipit bibendum purus augue cum         |dolores sea        |graecis voluptaria salutatus
+iisque phasellus                          |                   |inciderint vix              
+                                          |                   |mea                         
+antiopam porta omittantur mazim erroribus |option himenaeos   |consul nobis contentiones   
+                                          |aptent oporteat    |convenire dicam             
+                                          |                   |decore                      
+mi referrentur conclusionemque hac dapibus|iisque deterruisset|verear doming commune       
+duo tractatos                             |simul alienum      |                            
+autem                                     |                   |                            
+```
+
+### Blocked Text
+
+#### Kotlin API
+
+#### Java API
+
+## Issues
+
+[GitHub: Issues](https://github.com/NexusSandbox/inevitable/issues)
```

Comment:
Could favor issue templates as a transition (for the future): https://help.github.com/en/articles/creating-issue-templates-for-your-repository

Reply:
Yeah, that was a future update. For now, focusing on functionality.